src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -64,13 +64,16 @@ class NativeInterface
 
 
 
-
+        private Configuration _configuration;
 
         private bool CanRunOnBackgroundThread;
 
         private static bool Unity2019OrNewer;
         private Thread MainThread;
 
+        private bool _sessionCallbackSet;
```

Comment:
Picky as I know this just internal naming, but `_registeredForSessionCallbacks` used in the Cocoa implementation sounds better to me.